### DISCUSSION TITLE
Weekly Status Report: Week 19 2022

### DISCUSSION THREAD

**Discussion by cwayne18** [k3s-io/k3s#5563] [Weekly Status]
# Weekly Report
Weekly status report for k3s-io/k3s Week #19 2022

## Weekly Stats
| | Opened this week| Closed this week|
|--|---|-----|
|Issues|  10 |  18|
|PR's|  5 |  2|

|  |  |
|--|--|
| New stars | 152|
| New forks | 14|

## PR's Closed
* [#5555](https://github.com/k3s-io/k3s/pull/5555):  [Release-1.22] Add systemd cgroup controller support 
* [#4761](https://github.com/k3s-io/k3s/pull/4761):  Kubernetes 1.24

## PR's Opened
* [#5560](https://github.com/k3s-io/k3s/pull/5560):  Remove objects when removed from manifests
* [#5555](https://github.com/k3s-io/k3s/pull/5555):  [Release-1.22] Add systemd cgroup controller support 
* [#5554](https://github.com/k3s-io/k3s/pull/5554):  Bump dynamiclistener to v0.3.3
* [#5552](https://github.com/k3s-io/k3s/pull/5552):  Add cli flag for flannel wireguard mode
* [#5548](https://github.com/k3s-io/k3s/pull/5548):  Remove errant unversioned etcd go.mod entry

## Issues Opened
* [#5562](https://github.com/k3s-io/k3s/issues/5562):  Create by default `traefik` `ingressClass` when creating a cluster.
* [#5561](https://github.com/k3s-io/k3s/issues/5561):  Agent Nodes Saying Not Ready
* [#5559](https://github.com/k3s-io/k3s/issues/5559):  Objects removed from a manifest file aren't removed from the cluster
* [#5558](https://github.com/k3s-io/k3s/issues/5558):  The connection to the server localhost:8080 was refused - did you specify the right host or port?
* [#5557](https://github.com/k3s-io/k3s/issues/5557):  Validate Oracle Linux 8.5
* [#5556](https://github.com/k3s-io/k3s/issues/5556):  failed to setup network for sandbox
* [#5553](https://github.com/k3s-io/k3s/issues/5553):  [Release-1.22] - containerd to use the systemd driver
* [#5551](https://github.com/k3s-io/k3s/issues/5551):  Inclusion of 'non-default feature-gate enabled' features, particularly from kubernetes 1.24 onwards.
* [#5550](https://github.com/k3s-io/k3s/issues/5550):  [feat] upgrade CRD apiversion from beta1 to v1
* [#5549](https://github.com/k3s-io/k3s/issues/5549):  K3s pod not able to access internet which is in different network

## Issues Closed
* [#5556](https://github.com/k3s-io/k3s/issues/5556):  failed to setup network for sandbox
* [#5551](https://github.com/k3s-io/k3s/issues/5551):  Inclusion of 'non-default feature-gate enabled' features, particularly from kubernetes 1.24 onwards.
* [#5550](https://github.com/k3s-io/k3s/issues/5550):  [feat] upgrade CRD apiversion from beta1 to v1
* [#5528](https://github.com/k3s-io/k3s/issues/5528):  k3s install fails (Ubuntu 21.10 desktop)
* [#5514](https://github.com/k3s-io/k3s/issues/5514):  Ability to add dual-stacking to existing clusters
* [#5476](https://github.com/k3s-io/k3s/issues/5476):  Cut v1.23.6+k3s1
* [#5473](https://github.com/k3s-io/k3s/issues/5473):  Cut v1.22.9+k3s1
* [#5471](https://github.com/k3s-io/k3s/issues/5471):  Cut v1.21.12+k3s1
* [#5299](https://github.com/k3s-io/k3s/issues/5299):  Master Down, Entire Cluster Down
* [#4912](https://github.com/k3s-io/k3s/issues/4912):  [Epic] Create CLI to automate parts of release processes
* [#4332](https://github.com/k3s-io/k3s/issues/4332):  Traefik /ping 404 not found on new install
* [#4327](https://github.com/k3s-io/k3s/issues/4327):  Resource temporarily unavailable: Due to logs filling up /run/k3s/containerd/io.containerd.runtime.v2.task/k8s.io
* [#4322](https://github.com/k3s-io/k3s/issues/4322):  K3S with Hardening Ubuntu (20.04)
* [#4187](https://github.com/k3s-io/k3s/issues/4187):  Unable to start k3s service in the master node
* [#4150](https://github.com/k3s-io/k3s/issues/4150):  [Epic] KDM Automation
* [#3758](https://github.com/k3s-io/k3s/issues/3758):  [release-1.21] Imported images are not associated with k3s lease
* [#2636](https://github.com/k3s-io/k3s/issues/2636):  [epic] - Improved and Manual Cert Rotation
* [#1447](https://github.com/k3s-io/k3s/issues/1447):  How can I use the cloud-controller-manager for aws?

## Community PRs Closed
None

## Community PRs Opened
* [#5552](https://github.com/k3s-io/k3s/pull/5552):  Add cli flag for flannel wireguard mode
* [#5548](https://github.com/k3s-io/k3s/pull/5548):  Remove errant unversioned etcd go.mod entry

**Comment by pcgeek86** [comment]
Looks like 1.24 was merged 5 days ago? I'm excited for the 1.24 release! K3S is my favorite distribution for self-hosted Kubernetes!
